cephprimarystorage/cephprimarystorage/cephdriver.py

```
@@ -33,6 +33,10 @@ def clone_volume(self, cmd, rsp):
         rsp.installPath = dst_path
         return rsp
 
+    def resize_volume(self, cmd, rsp):
+        linux.qemu_img_resize('rbd:%s/%s' % (pool, image_name), cmd.size, 'raw', cmd.force)
```

Ruff (0.8.2)

`linux` may be undefined, or defined from star imports

(F405)